plugins/scale/scale.cpp

```diff
@@ -860,7 +885,9 @@ class wayfire_scale : public wf::per_output_plugin_instance_t,
         {
             std::sort(views.begin(), views.end(), [=] (wayfire_toplevel_view a, wayfire_toplevel_view b)
             {
-                return wf::get_focus_timestamp(a) > wf::get_focus_timestamp(b);
+                /* note: avoid focusing minimized views if possible */
+                return (a->minimized < b->minimized) ||
```

Comment:
I would split this into an if - else for better readability.

Reply:
I've come up with an alternative, since it's also a bit weird to compare bools this way.